### PR TITLE
OOP_2 hometask is completed

### PR DESCRIPTION
**Pull request by NadzeyaKiz**
The homework is ready to be checked

### CODE REVIEW

File: Code/CSharpOOP2/Homework.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP2
+{
+    public class Table
```
Comment (dnnyhlwdplc) [inline Code/CSharpOOP2/Homework.cs]: please check that class name in solution explorer and solution are the same :)

File: Code/CSharpOOP2/Homework.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            {
+                Console.WriteLine("Table {0} is avalible.", Name);
+            }
+            else
+            {
+                Console.WriteLine("Product {0} is temporarily out of stock.", Name);
+            }
+        }
+        //Class constructor Task_6
+        public Table(string name, int quantity, double price, string brand, bool isAvailable)
```
Comment (dnnyhlwdplc) [inline Code/CSharpOOP2/Homework.cs]: just a minor comment: constructors should be above fields and fields should be above methods according to style code

File: Code/CSharpOOP2/Homework.cs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        }
+        // Constructor initializing 2 fields (Task_08)
+        public Table(string name, int quantity)
+        {
+            Name = name;
+            Quantity = quantity;
+        }
+
+        // Method with private modifier (Task_11)
+        private void PrivateMethod()
```
Comment (dnnyhlwdplc) [inline Code/CSharpOOP2/Homework.cs]: there should be method that works with your object

File: Code/CSharpOOP2/Homework.cs [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        {
+            Console.WriteLine("Private Method");
+        }
+
+        // Method with internal modifier (Task_11)
+        internal void InternalMethod()
```
Comment (dnnyhlwdplc) [inline Code/CSharpOOP2/Homework.cs]: same

File: Code/CSharpOOP2/Program.cs [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+#region (Task_13)
+Table table5 = new Table()
+{
+    Name = "Rainbow",
+    Quantity = 10,
+    Brand = "Test",
+    IsAvailable = true
+};
+// Вывод значения поля Price
+Console.WriteLine($"Значение поля Price: {table5.Price}");
```
Comment (dnnyhlwdplc) [inline Code/CSharpOOP2/Program.cs]: English please :)

**Review by dnnyhlwdplc — APPROVED** [review]

File: Code/CSharpOOP2/Homework.cs (hunk 1 above)
Comment (NadzeyaKiz) [reply Code/CSharpOOP2/Homework.cs]: done

File: Code/CSharpOOP2/Homework.cs (hunk 2 above)
Comment (NadzeyaKiz) [reply Code/CSharpOOP2/Homework.cs]: done

File: Code/CSharpOOP2/Homework.cs (hunk 3 above)
Comment (NadzeyaKiz) [reply Code/CSharpOOP2/Homework.cs]: Changed

File: Code/CSharpOOP2/Homework.cs (hunk 4 above)
Comment (NadzeyaKiz) [reply Code/CSharpOOP2/Homework.cs]: Changed

File: Code/CSharpOOP2/Program.cs (hunk 5 above)
Comment (NadzeyaKiz) [reply Code/CSharpOOP2/Program.cs]: done

**Pull request merged**